### ISSUE TITLE
Better Lightbox implementation

### ISSUE THREAD

**Issue by svengato** [PeanutBase/jekyll-peanutbase#35]
Lightbox is the JavaScript library that displays the large image and obscures the background when you click on a thumbnail image. The [currently used implementation](https://jekyllcodex.org/without-plugin/lightbox/) is pretty minimal. It would be nice to have some of the functionality of the old version, like arrows to scroll to the previous/next image in the set.

**Comment by svengato** [comment]
I was able to tweak the current Lightbox JavaScript and CSS code to make arrow navigation work. Still to do:
- [x] You currently have to click the arrows by hand, but it should be possible to use the left/right arrow keys as well (also Esc to get out of Lightbox and go back to the current image set page). (We decided this is a low priority.)
- [x] Need to further examine the JavaScript for possible unintended side effects (it should limit itself to the large images from the current image set, compare to a clone of the image array instead of the image array itself, etc).
